spv/spv_test.go

```diff
@@ -56,7 +56,6 @@ func TestExecuteSimplifiedPaymentVerification_CorruptedTransaction_ReturnError(t
016eaca47b94121033733254e143eba5ee6989fe898647d4e6170c0d82faa72fbce1c1e701960edb9ffffffff0264000000000000001976a914ddfbaa2cd75b86cf24135603b5ddd2b1968bc62d88ac8e010000000000001976a9148360b65a14e83a68aef2fae3f9cf472a90dd4da488ac000000000001000000029fc71a2f88088174c4d0e0348e7087c9b355a4e93e5b2ec4a5f41fdd69272f86010000006a47304402203a5c238e8ceccff7b7db05a4dba524f6694599014dbac5ae92dc0c62635c8763022078cab36b04a16eeead80fa49dd4d1c22f14d8084c78977917cfe76fee9d95a84412102993285bd72def003516c414b4fb3c3718d5d003c90f08c75bd6eecc08910585fffffffff396915fa4cff57573f0e80b1d585d5a4f7692984e51577c8ea82e1d88f8404ce060000006a47304402205ba90f2ed3f70c779771185a473e3b1844ad0f8f86bddf39148ad277c9a2298902203df352b710b05501cc3d773e366854770038f9dea65c046b666fd28b124f303d412103c335fc77d28a0181bec42cd7afda547a4d524c5b1f04a7d9d0ec76f1217a0e5affffffff03f4010000000000001976a91494c77286c0552398575ea741337dc6143c7978ec88ac2b050000000000001976a91418d9eaafcf7d8e8ba29ac102ed19b9b3808376a088ac0f4e0000000000001976a914555b7d05b5327c78c7bf7f8748c1b3ce3552168888ac00000000000100000001f8aea749791d0907e46bbb7df82dc52f84a9b83434e563e3a6496b95239fc08d000000006b483045022100a50ebc1429285edf0d39d77f0ab49e68eaf6e8f97d727671fcc51763235ca39a02200252a185b70fb5ac893ad32e8ffe1e8ade9d3d8993f8db4a8cb27dc6e32c18664121037b231d9fdd2f7a41181de9a14ae11a10b7332b1fdb5526c1a0060eb964f111d6ffffffff0296000000000000001976a914ddfbaa2cd75b86cf24135603b5ddd2b1968bc62d88ac5d010000000000001976a9148360b65a14e83a68aef2fae3f9cf472a90dd4da488ac0000000000010000000183d98243af506e20461097054c6692a09502ee7a99eba67cd20b0d5b66d081f0010000006a473044022055e323fc40f05fdeb836fadf19003d91f08abef97526e04f4c976156faedc83c022050f36b4d23329fa9d6f37bfee952ae3a0bf266573fc19a51ab42595ee674a26a4121033733254e143eba5ee6989fe898647d4e6170c0d82faa72fbce1c1e701960edb9ffffffff0264000000000000001976a914ddfbaa2cd75b86cf24135603b5ddd2b1968bc62d88ac29010000000000001976a9148360b65a14e83a68aef2fae3f9cf472a90dd4da488ac000000000001000000032a796907de322523a34e801e304ea43b6938cfa5afe3ff4fea7fbc34bd52164c010000006a47304402203b0ef0d3ccba1cc3dcb9dbccdc49a7eca5a7f887564960050110a7a979cd8026022045eac010d965623c002a06946c9e9fe2a17ca0504d5eb90b67bc73a8e8904c624121033733254e143eba5ee6989fe898647d4e6170c0d82faa72fbce1c1e701960edb9ffffffff73bd2e109d1e01d9c5828e15b7e3212e54dd3d9558ceae6514c97e07ad275110000000006b483045022100b05c6873b1161dc0ea574cfbb2fdb3843782f2f7ea3c136a7086b69df8a8e5ed022019c3d8cb94c75c4c9a6acc44c9c7809a99e5cd6427b1ab5fcdd6a7dc1527331b4121033733254e143eba5ee6989fe898647d4e6170c0d82faa72fbce1c1e701960edb9ffffffff95e93ce1a376cfd842ffbd849f0ee37330d6b62a51cddfacfbed9c1a4d49cce7000000006a473044022044c557f9fe6c42e3c22e9ee898d5ae2adf11f08bdc2cee6d06954238c60b95a1022072a5b9de839bf851958b6b2aa7e8c28c43f8275333e0cf6cc272d75ec4740e7c41210275aaa422bc13c303b70026b3fd1fff37c199e5e98bd0a429922217f9b861870effffffff0220030000000000001976a914ddfbaa2cd75b86cf24135603b5ddd2b1968bc62d88ac92000000000000001976a9148360b65a14e83a68aef2fae3f9cf472a90dd4da488ac0000000000"
```

Comment:
Why?

Reply:
This value was unused and linter job failed, because of it